lib/potluck.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
+require './lib/dish'
+
+class Potluck
+
+  attr_reader :date,
+              :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category)
+    result = []
+    @dishes.each do |dish|
+      if dish.category == category
+        result << dish
+      else
+        dish
+      end
```

Comment:
Don't need this else block. Really what you're saying here is don't do anything with the dish if the category doesn't match. Possible to have an if without an else.